app/src/main/res/values/strings.xml

```diff
@@ -139,6 +140,12 @@
     <string name="app_components_controlItem_divider_label">Divider</string>
     <string name="app_components_controlItem_inverted_label">Inverted</string>
 
+    <!-- Components: divider -->
+    <string name="app_components_divider_label">Divider</string>
+    <string name="app_components_divider_description_text">A divider visually structures an interface by clearly separating content sections. It helps to improve readability and content organization without introducing a strong hierarchy like a heading or a container would.</string>
+    <string name="app_components_divider_horizontal_label">Horizontal divider</string>
```

Comment:
We wrote `app_components_checkbox_checkboxItem_label` for the checkbox item variant. Does that mean that we should write `app_components_divider_horizontalDivider_label` here?